mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule Broadway.MixProject do
       extras: [
         "guides/examples/Amazon SQS.md",
         "guides/examples/RabbitMQ.md",
-        "guides/examples/Custom producers.md",
+        "guides/examples/Custom Producers.md",
```

Comment:
this wasn't caught before because looks like someone is using a case-insensitive filesystem ;-)